src/adapter/src/catalog/storage.rs

```diff
@@ -148,6 +148,12 @@ async fn migrate<S: Append>(
                 },
                 IdAllocValue { next_id: 1 },
             )?;
+            txn.id_allocator.insert(
+                IdAllocKey {
+                    name: STORAGE_METRICS_ID_ALLOC_KEY.into(),
```

Comment:
There was a request to change this and the string it maps to to use `usage` instead of `metrics` in the name. Should probably do that while we're squashing migrations.
aka -- `STORAGE_USAGE_ID_ALLOC_KEY`, `storageusage`

Reply:
I sent #14215 to get ahead of this, since we're on the clock to get this all in by Tuesday.